Splunk Phantom/DomainTools_Iris_Malicious_Tags/README.md

```diff
@@ -1,9 +1,14 @@
 ## DomainTools Iris Malicious Tags Playbook
 
-This playbook retrieves the Iris Investigate profile of a domain and automatically flags domains that have been tagged as malicious in DomainTools Iris. The list of malicious tags are configurable by the analyst and can be kept up to date for automation.
+Retrieves the Iris Investigate profile of a domain and automatically flag domains that have been tagged as malicious in DomainTools Iris. The list of malicious tags are configurable by the analyst and can be kept up to date for automation.
```

Comment:
I moved these to a google doc so that I could get spell check. I really messed them up, I'll fix it. 
Thank you for looking at this @mccarthym, a lot of good catches! I'll let you know when I get it fixed up. Wow me.

Reply:
This file has been fixed